gitless/cli/helpers.py

```diff
@@ -129,6 +129,9 @@ def process_paths():
                 self.skip_dir_cb(curr_dir_rel)
               dirs[:] = []
               continue
+            if not fps:
+              open(curr_dir + "/.glkeep.", "a").close()
```

Comment:
The `.` at the end of the filename is an attempt of fixing a [problem on Windows](https://ci.appveyor.com/project/spderosso/gitless/build/1.0.17/job/393n9lk1ugh86k8n). Reference: https://gist.github.com/ozh/4131243#1-rename